packages/libs/sdk/src/api/controllers/nfts.ts

```diff
@@ -207,10 +215,6 @@ export class NftsController {
       variables: queryVariables,
     });
 
-    if (!trendingCollections?.trendingCollections?.length) {
-      return { results: [], pageInfo: emptyPageInfo };
-    }
-
```

Comment:
This block isn't actually doing anything now, there's no way to trigger it as trending collections should always be returned from the API and if it's not then the empty results from the API will be returned